cogs/trading.py

```diff
@@ -355,7 +355,7 @@ async def cancel(self, ctx):
     @checks.has_started()
     @commands.guild_only()
     @trade.command(aliases=("c",))
-    async def confirm(self, ctx):
+    async def confirm(self, ctx, user: discord.Member):
```

Comment:
Remove user argument

Reply:
But if I remove that, then how to get the trade as it requires the `user` argument
```suggestion
     trade = self.bot.trades[user.id]
```